Updated Models section (#1025)
ACCESS-Hive/access-hive.github.io: docs/models/index.md

# Models
<div class="card-container">
    <!-- ACCESS Models -->
    <a href="access_models/" class="horizontal-card">
        <div class="card-image-container">
            <img class="img-contain white-background" src="/assets/ACCESS-MODEL.png" alt="ACCESS Models">
        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS Models</span>
            <span>

>                 ACCESS models are computer programs which can be used to simulate past or future conditions of the Earth System, as well as carrying out idealised experiments.

Author: Good point!

            </span>
        </div>
    </a>
    <!-- Model Components -->
    <a href="model_components/" class="horizontal-card">
        <div class="card-image-container">
            <img class="img-contain white-background" src="/assets/component-logos/all_components.png" style="padding: 0.2em;" alt="Model Components">
        </div>
        <div class="card-text-container">
            <span class="bold">Model Components</span>
            <span>

>                 ACCESS model components are computer programs representing different chemical, physical or biological parts of the Earth System. These components are interconnected together to form ACCESS models.

Reviewer: Same comment as above. A possible option below:

```suggestion
                Overview of the Earth system components that make up the ACCESS models
```

Author: Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852
